journal/modernized/04-chapter-3.adoc

```diff
@@ -122,13 +122,14 @@ our hearts were truly opened one to another, and glad to hear of Truth`'s prospe
 After we had visited Munster, we returned to the province meeting in Leinster,
 and visited the counties of Carlow, Wexford, and Wicklow.
 Then finding my spirit clear, I returned to Dublin, took shipping for Whitehaven,
-and at the latter end of the eleventh month, 1698, I landed safe, and`' returned home,
+and at the latter end of the eleventh month, 1698, I landed safe, and returned home,
 staid some time, and was very diligent in attending meetings.
 
 [.offset]
 +++[+++The following Epistle to Friends in America,
 which is not inserted either in the MS or the former edition of this Journal,
 is recorded in Gough`'s History of Friends.]
+//What is MS on line 130? The editor pointed it out.  Is there a way to clarify?
```

Comment:
I'm pretty certain `MS` there means "manuscript", and could probably be changed to read `manuscript` for easier comprehenstion.

And, nice use of a comment! 😄